src/app/admin/users/UserTablePage.tsx

```diff
@@ -83,6 +83,12 @@ const UserTablePage = () => {
       console.log("Updating data:", updatedData);
       try {
         const response = await client.models.User.update(updatedData);
```

Comment:
You can remove this as it gets accomplished in the AddUserToGroup backend function. Otherwise good!

Reply:
@ideen1 the addUserToGroup function only edits the roles not any other fields

This client.models.User.update(updatedData) does all the rest of the data (firstname, lastname, etc)

Reply:
Oh I didn't realize this part of the code was responsible for the whole thing. Since we are using RHF, can we make it so if that field is touched then we call the backend function, otherwise don't. And if firstname/lastname are changed then we update the table, otherwise no update?

Reply:
ok

Reply:
I fixed it